core/src/main/java/io/debezium/connector/cassandra/metrics/CassandraSnapshotMetricsMXBean.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.cassandra.metrics;
+
+import java.util.Map;
+
+/**
+ * MXBean interface for Cassandra snapshot metrics to ensure JMX compliance.
+ */
+public interface CassandraSnapshotMetricsMXBean {
```

Comment:
Why it is not possible to re-use `io.debezium.pipeline.metrics.traits.SnapshotMetricsMXBean`?